x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_event_log_list_status_filter.tsx

```diff
@@ -7,11 +7,11 @@
 
 import React, { useState, useCallback } from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
+import { RuleAlertingOutcome } from '@kbn/alerting-plugin/common';
 import { EuiFilterButton, EuiPopover, EuiFilterGroup, EuiFilterSelectItem } from '@elastic/eui';
-import type { EcsEventOutcome } from '@kbn/core/server';
 import { RuleEventLogListStatus } from './rule_event_log_list_status';
 
-const statusFilters: EcsEventOutcome[] = ['success', 'failure', 'unknown'];
+const statusFilters: RuleAlertingOutcome[] = ['success', 'failure', 'warning'];
```

Comment:
I am not sure about removing "unknown" :)
I know currently we never set rule execution result "unknown" but don't know if it was always like this. 
Are we sure that none of the users has "unknown" in their ES? Maybe @ymao1 knows :)

Reply:
Resolved here d48c414689826c3e51b7c763acd05b695395968a